### PR TITLE
triggered incidents view, incidents limit modified to show all incidents (high/low)

### PR DESCRIPTION
**Pull request by supreeth7**

### CODE REVIEW

File: pkg/ui/seeders.go [hunk 1]
```diff
@@ -61,6 +63,13 @@ func (tui *TUI) SeedIncidentsUI() {
 	var incidentsData [][]string
 
 	tui.IncidentOpts.Statuses = []string{constants.StatusTriggered}
+
+	// Override incidents limit when viewing triggered incidents assigned to self
+	if tui.AssginedTo == tui.Username {
+		tui.IncidentOpts.Limit = 25
```
Comment (T0MASD) [inline pkg/ui/seeders.go:69]: Should this be set as const var instead of hard code?

**Review by T0MASD — COMMENTED** [review]
Looks good just one point in the comment

File: pkg/ui/seeders.go (hunk 1 above)
Comment (supreeth7) [reply pkg/ui/seeders.go]: Right, I've refactored the variable to a constant

**Review by T0MASD — APPROVED** [review]
/approved
/lgtm
/label tide/merge-method-squash

**Pull request merged**